source/fem-dem/cfd_dem_coupling.cc

```diff
@@ -1406,6 +1406,9 @@ CFDDEMSolver<dim>::dem_setup_contact_parameters()
           break;
         }
     }
+
+  // Initialize the total contact list counter
+  contact_search_total_number = 0;
```

Comment:
This was not initialized, so the first number in the stats was wrong (and kinda random)